src/react/hooks/__tests__/useQuery.test.tsx

```diff
@@ -2092,6 +2092,200 @@ describe("useQuery Hook", () => {
       unmount();
       result.current.stopPolling();
     });
+
+    describe("should stop refetching if `shouldRefetchOnPolling` is passed at", () => {
+      beforeEach(() => {
+        jest.useFakeTimers();
+      });
+
+      afterEach(() => {
+        jest.runOnlyPendingTimers();
+        jest.useRealTimers();
+      });
+
+      it("global level", async () => {
```

Comment:
```suggestion
      it("when defined as a global default option", async () => {
```

Just trying to make it more obvious what is being tested here 🙂